test/Utility-test.js

```diff
@@ -0,0 +1,116 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+import allSleepData from '../data/sleep';
+import sleepData from '../data/sleep-test-data';
+import userData from '../data/users-test-data';
```

Comment:
Thank you for building out this test suite!  I think that just using the sleepData for the testing is ok even though these methods will be used on activity and hydration data as well.  All the methods accept whatever data is passed through so I do not think we need to overload the test suite by including all of them.   

@brittanystoroz @khalidwilliams   Do you agree?

Reply:
@hfaerber @Dunn-Austen I don't think it would hurt to be explicit with each dataset here. Especially since some methods are meant to take in parameters for individual stats by date, and that could look different depending on what dataset you're working with.